src/controllers/StorageController.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  JsonController,
+  Get,
+  Post,
+  QueryParam,
+  UploadedFile,
+  BadRequestError,
+  UseBefore,
+} from 'routing-controllers';
+import { AppUserService, AppUserServiceImpl, StorageService, StorageServiceImpl } from '@Services';
+import multer from 'multer';
+import { OfficerAuthMiddleware } from '@Middlewares';
+import { OpenAPI } from 'routing-controllers-openapi';
+
+const fileFilter = (req: Express.Request, file: Express.Multer.File, cb: Function) => {
+  if (
+    file.mimetype.includes('image') ||
+    file.mimetype.includes('pdf') ||
+    file.mimetype.includes('word')
+  ) {
+    cb(null, true);
+  } else {
+    console.log('Invalid file type');
+    cb(null, false);
+  }
+};
+
+const fileUploadOptions = {
+  storage: multer.memoryStorage(),

```

Comment:
could we add a short comment on this? sounds scary that we're using memoryStorage (but probably not form data)

Reply:
done